admin_pages/registration_form/templates/questions_main_meta_box.template.php

```diff
@@ -316,21 +316,21 @@ class="QSO_order"
                                                 <input type="text"
                                                        class="option-value regular-text"
                                                        name="question_options[<?php echo $count ?>][QSO_value]"
-                                                       value="<?php echo esc_attr($option->get_pretty('QSO_value')) ?>"
+                                                       value="<?php echo esc_attr($option->f('QSO_value')) ?>"
```

Comment:
Are you sure this will work ok ? Because `f()` usually echoes out.

Reply:
Yeah it works, but will remove the additional echo. Nice catch.